react/features/speaker-stats/components/SpeakerStats.js

```diff
@@ -276,27 +170,20 @@ class SpeakerStats extends Component<Props, State> {
      * @protected
      */
     _onSearch(criteria = '') {
-        this.setState({
-            ...this.state,
-            criteria: escapeRegexp(criteria)
-        });
+        APP.store.dispatch(initSearch(escapeRegexp(criteria)));
```

Comment:
You should get the dispatch from the props. Usage of APP... is only for the non-react code ...